_includes/main.scss

```diff
@@ -706,4 +706,12 @@ img {
   font-size: 1.25rem;
   padding-bottom: 1rem;
   padding-top: 0.5rem;
+}
+[id]::before { 
+  display: block; 
+  content: " "; 
+  margin-top: -60px; // height of header bar
+  height: 60px; 
+  visibility: hidden; 
+  pointer-events: none;
```

Comment:
<img width="374" alt="Screenshot 2024-04-22 at 14 00 53" src="https://github.com/techworkersco/twc-site/assets/7111514/36ba070f-b61c-46cd-8ab9-a667a330268e">
[Before](https://deploy-preview-361--techworkersco.netlify.app/community-guide/#membership) and [after](https://deploy-preview-362--techworkersco.netlify.app/community-guide/#membership ) the anchor tags were not visible.
<img width="583" alt="Screenshot 2024-04-22 at 14 01 08" src="https://github.com/techworkersco/twc-site/assets/7111514/7bfa07b4-cd29-4b0e-acb8-b979e71759ae">